unit_test/test_options.py

```diff
@@ -0,0 +1,85 @@
+from flask import json
+
+from unit_test.helpers import create_dummy_user, create_headers, create_dummy_box, create_dummy_option
+
+
+def test_create_new_option(client):
+    # create header
+    token = create_dummy_user(username="mia", password="abc")
+    headers = create_headers(token)
+    box_id = create_dummy_box(1, "Minerva Feast", "06-02-2021")
+
+    # invalid input
+    data1 = {"name": 1, "description": "Vietnamese food"}
+    response = client.post(f"/boxes/{box_id}/options", json=data1, headers=headers)
+    assert response.status_code == 400
+    assert {"name": ["Not a valid string."]} == json.loads(response.data)
+
+    # invalid input
+    data1 = {"name": "Ngon ngon", "description": 1}
+    response = client.post(f"/boxes/{box_id}/options", json=data1, headers=headers)
+    assert response.status_code == 400
+    assert {"description": ["Not a valid string."]} == json.loads(response.data)
+
+    # invalid box_id
+    data1 = {"name": "Ngon ngon", "description": "Vietnamese food"}
+    response = client.post("/boxes/10/options", json=data1, headers=headers)
+    assert response.status_code == 404
+    assert {'message': 'Cannot find the wizbox'} == json.loads(response.data)
+
+    # valid input
+    data1 = {"name": "Ngon ngon", "description": "Vietnamese food"}
+    response = client.post(f"/boxes/{box_id}/options", json=data1, headers=headers)
+    assert response.status_code == 200
+    expected_response = {
+        "name": "Ngon ngon",
+        "description": "Vietnamese food",
+        "id": 1
+    }
```

Comment:
Great tests to check the validity of the request body.

To make tests more robust you should add tests for:

1. Empty JSON object as the request body.
2. No request body (send no object in the request body).

Reply:
✅